dcrtimed/dcrtimed.go

```diff
@@ -187,10 +193,23 @@ func convertDigests(d []string) ([][sha256.Size]byte, error) {
 	return result, nil
 }
 
+func (d *DcrtimeStore) proxyWalletBalance(w http.ResponseWriter, r *http.Request) {
+	var apiToken string
+	queryTokens := r.URL.Query()["apitoken"]
+	if len(queryTokens) > 0 {
+		apiToken = queryTokens[0]
+	}
+
+	// Only forward the first apitoken query string parameter
```

Comment:
isn't apitoken required?

Reply:
yes, the backend checks this and the 401 gets pushed back to the client in `sendToBackend`